src/Common/Button/Button.jsx

```diff
@@ -5,7 +5,7 @@ import "./Button.scss";
 
 import cx from "classnames";
 
-function Button({ label, onClick, type, className, inline }) {
+function Button({ label, onClick, type, className, inline, social }) {
```

Comment:
what is the "social" thing for?

Reply:
I wanted to do something else, then changed my mind..